assets/js/builder/Views/Question.js

```diff
@@ -84,7 +84,7 @@ define( [
 		/**
 		 * Compiles the template and renders the view
 		 * @return   self (for chaining)
-		 * @since    3.16.0
+		 * @since    [version]
```

Comment:
```suggestion
		 * @since 3.16.0
		 * @since [version] What I did.
```

Then also remove the `@version` tag, and move the `@return` below the `@since` block, leaving an empty line, also remove useless spaces and terminate lines with a fullstop, exactly the same thing we do with PHP functions.

Same rule, we don't want to update ALL the documentation blocks of a file, but when we touch a documentation block to change something, well let's update it with the new documentation standards.